scripts/debian/falco.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Falco: Container Native Runtime Security
+
+[Service]
+Type=simple
+User=root
+ExecStart=/usr/bin/falco --pidfile=/var/run/falco.pid
+UMask=0077
+TimeoutSec=30
+RestartSec=15s
+#Restart=always
+Restart=on-failure
+NoNewPrivileges=yes
+PrivateTmp=true
+ProtectHome=read-only
+ProtectSystem=full
+ProtectKernelTunables=true
+RestrictRealtime=true
+#RestrictAddressFamilies=AF_INET AF_INET6 AF_UNIX
+RestrictAddressFamilies=~AF_PACKET
+SystemCallFilter=~@debug @mount @cpu-emulation @obsolete @privileged
+# FIXME!
+# PANIC: unprotected error in call to Lua API (runtime code generation failed, restricted kernel?)
+# https://www.freelists.org/post/luajit/luajit-crashes-with-grsec-kernel,1
+# MemoryDenyWriteExecute=true
+# PrivateMounts=true
```

Comment:
Could you elaborate more on that?

Reply:
I'm referencing this PR https://github.com/falcosecurity/falco/pull/989

I'm not sure which part should be preserved or should I directly refer to the issue description https://github.com/falcosecurity/falco/issues/1265#issue-640658962?

Reply:
Not a systemd expert. But for sure I'd avoid leaving those comments if we are not sure about that. 

/cc @fntlnz 
/cc @leodido 
/cc @kris-nova 

WDYT?